### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by yogesh-**
- Installed mockbee backend
- setting up the homepage

image

### CODE REVIEW

File: src/pages/home/home.jsx [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+          <div class="search-bar flex-row">
+            <input type="text" placeholder="Search" />
+            <span class="material-icons-outlined search-icon">search</span>
+          </div>
+
+          <div class="nav-items">
+            <ul class="flex-row right-icons">
+              <li>
+                <a href="/html/login.html">
```
Comment (tushargoyal812) [inline src/pages/home/home.jsx:24]: suggestion:you can make navbar component so your code looks more clean.

Comment (yogesh-) [reply src/pages/home/home.jsx:24]: yes, will break the homepage into several components.

**Review by akash-1318 — COMMENTED** [review]
In case you are going to use components then it's good but right now i can see you are not so please use components for different sections.

File: src/pages/home/home.jsx [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+                  </button>
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        {/* <!-- footer --> */}
+
+        <div class="footer-home flex-col">
```
[the remark this answers is not in the document]
Comment (yogesh-) [reply src/pages/home/home.jsx:198]: pushing in the next pr

**Pull request merged**